src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -428,6 +432,7 @@ private Msg getEngineExceptionMsg(Exception e) {
   @APIResponse(responseCode = "200", description = "Policy deleted")
   @APIResponse(responseCode = "404", description = "Policy not found")
   @APIResponse(responseCode = "403", description = "Individual permissions missing to complete action")
+  @Parameter(name = "id", description = "UUID of the policy", schema = @Schema(type = SchemaType.STRING))
```

Comment:
Could you remove the `schema` from here (and the others params that are already a `@PathParam`), the schema seems to be inferred from the variable type.

In this case, you are changing the type from `Uuid` to `String`.

For ui-frontend, it means this change:

```diff
 export interface UsePostPoliciesValidateNameParams {
     body?: string;
-    id?: schemas.Uuid;
+    id?: string;
 }
```

Reply:
I see that you removed the `@Schema` from the other places, but here, could you do the same here? :)